CanvasCore/CanvasCore/WhizzyWig/CanvasWebView.swift

```diff
@@ -200,8 +200,7 @@ public class CanvasWebView: WKWebView {
     
     @objc public func htmlContentHeight(completionHandler: @escaping (CGFloat) -> Void) {
         evaluateJavaScript("document.documentElement.scrollHeight") { result, error in
-            guard let height = result as? NSNumber else { completionHandler(0.0); return }
-            completionHandler(CGFloat(height))
```

Comment:
This initializer is deprecated, just cast straight to `CGFloat` instead of through `NSNumber`.